core-services/release-controller/_releases/release-ocp-4.17-multi.json

```diff
@@ -11,18 +11,24 @@
   "check": {},
   "publish": {},
   "verify": {
-    "multi-aws-ovn-upgrade": {
+    "e2e-aws-ovn-multi-x-ax": {
+      "maxRetries": 2,
+      "prowJob": {
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-multi-x-ax"
+      }
+    },
+    "e2e-aws-ovn-nightly-upgrade-multi-x-ax": {
       "optional": true,
       "prowJob": {
-        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-heterogeneous-upgrade"
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-upgrade-multi-x-ax"
       },
       "upgrade": true,
       "upgradeFrom": "Previous"
     },
-    "multi-aws-ovn-upgrade-from-previous-stable": {
+    "e2e-aws-ovn-minor-upgrade-multi-x-ax": {
       "optional": true,
       "prowJob": {
-        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-upgrade-from-stable-4.16-ocp-e2e-aws-ovn-heterogeneous-upgrade"
+        "name": "periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-upgrade-multi-x-ax"
```

Comment:
Yep, updating (but I reverted the filename changes, so its back to the long one... sadly.  I'll be doing a sweeping update to all job names since we're a little unwieldy currently across the board... hah...